### ISSUE TITLE
Plugin broken with Maven 3.9.0

### ISSUE THREAD

**Issue by cstamas** [teetrinkers/resolver-maven-plugin#2]
The plugin does not work with Maven 3.9,0+ (and later, like 4.0.0-alphas). Reason is simple: it relies on Maven 2 (!) behavior, where plexus-utils is "auto injected" to plugins as dependency, even without declaring them as such.

The used class from plexus-utils is `org.codehaus.plexus.util.StringUtils`.

How to fix it: just make plugin POM explicitly declare dependency on plexus-utils, like this:
```
      <dependency>
        <groupId>org.codehaus.plexus</groupId>
        <artifactId>plexus-utils</artifactId>
        <version>3.5.0</version> <!-- or any version plugin wants -->
      </dependency>
```

**Comment by teetrinkers** [comment]
Hi Tamas,

thank you for the pointer. I can confirm that the plugin does not work with Maven 4.0.0-alpha-4. I will look into it.

**Comment by teetrinkers** [comment]
Fixed in version 1.0.2.
